src/ucp/wireup/select.c

```diff
@@ -1196,7 +1195,29 @@ ucp_wireup_add_rma_bw_lanes(const ucp_wireup_select_params_t *select_params,
     bw_info.max_lanes         = context->config.ext.max_rndv_lanes;
```

Comment:
remove it, pls
this is not needed anymore as we set it below to `1` in case of RKEY_PTR is supported or/and to `context->config.ext.max_rndv_lanes`.

Reply:
fixed, was a leftover